### DISCUSSION TITLE
Enforce line break after heading in lint

### DISCUSSION THREAD

**Discussion by mastercoms** [remarkjs/remark#1118] [Q&A] [answered]
Hi, is there an available plugin for enforcing a line break after a heading like so:

Pass:
```
# Heading

Content
```

Fail:
```
# Heading
Content
```

I've been encountering problems with some parsers which do not work with the fail case, and tried searching around for this to no avail, maybe missing the right terminology or keyword

**Comment by Murderlon** [comment, marked as answer]
If you use `remark` it automatically adds a space to it ([example](https://codesandbox.io/s/jovial-ben-gkcbb9?file=/src/index.js)). If you use the CLI, make sure to use the `--output` flag.

**Comment by mastercoms** [comment]
Is there a remark-lint for this as well?

**Reply by wooorm** [reply]
These are called blank lines, so you can find the rule looking for that term: https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-missing-blank-lines.

A line break is a single one, this: -->
<--.

**Reply by mastercoms** [reply]
Ah thank you so much for the clarification and help!
